massa-execution-worker/src/tests/wasm/sources/datastore/main.ts

```diff
@@ -0,0 +1,19 @@
+import { print, generateEvent, getOpKeys, hasOpKey } from "@massalabs/massa-sc-std";
```

Comment:
missing doc explaining the role of the file

Reply:
done